swift/SampleApp/SampleApp/Mobility/Data/ITSMobilityService.swift

```diff
@@ -0,0 +1,89 @@
+/*
+ * Software Name : ITSClient
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * see the "LICENSE.txt" file for more details or https://opensource.org/license/MIT/
+ *
+ * Software description: Swift ITS client.
+ */
+
+import Foundation
+import ITSCore
+import ITSMobility
+
+actor ITSMobilityService: MobilityService {
+    private let mobility: Mobility
+    private var roadAlarmChangeObserver: RoadAlarmChangeObserver?
+    private var sendPositionTask: Task<Void, any Error>?
+
+    init() {
+        mobility = Mobility()
+    }
+
+    func start() async throws(MobilityError) {
+        // Update this to set your broker connection configuration
+        let host = ""
+        let port = 1883
+        let clientIdentifier = UUID().uuidString
+        let username: String? = nil
+        let password: String? = nil
+        let useSSL = false
+
+        let mqttClientConfiguration = MQTTClientConfiguration(host: host,
+                                                              port: port,
+                                                              clientIdentifier: clientIdentifier,
+                                                              userName: username,
+                                                              password: password,
+                                                              useSSL: useSSL)
+        let coreConfiguration = CoreConfiguration(mqttClientConfiguration: mqttClientConfiguration)
+        let mobilityConfiguration = MobilityConfiguration(coreConfiguration: coreConfiguration,
+                                                          stationID: 165_315,
```

Comment:
Maybe explain to users why this value must be used for station id?

Reply:
Just a constant random number ;)